### PR TITLE
matched 2 functions

### PR DESCRIPTION
**Pull request by Unnunu**

### CODE REVIEW

File: src/7bb60_len_41b0.c [hunk 1]
```diff
@@ -765,11 +763,23 @@ s32 collision_check_player_intersecting_world(s32 mode, s32 arg1, f32 arg2) {
...
+        s32 hitID, hitID2;
 
-        if (hitID >= 0) {
-            ret = hitID;
+        hitID = player_test_lateral_overlap(mode, gPlayerStatusPtr, &x, &y, &z, 0.0f, angle);
+
+        // required to match
+        if (hitID > 0 || hitID == 0) {
+            hitID2 = hitID;
+        } else if (hitID == -1){
```
Comment (nanaian) [inline src/7bb60_len_41b0.c:773]:
```suggestion
        } else if (hitID == -1) {
```

**Review by nanaian — CHANGES_REQUESTED** [review]
small nit; good work!

**Comment by nanaian** [comment]
@ethteck looks like Jenkins died, perhaps a retry is in order

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.06%) on `us`.

**Review by ethteck — APPROVED** [review]

**Pull request merged**